Yoast/javascript
# P3-170 Changes and additions needed for SEMrush integration feature

## Summary


This PR can be summarized in the following changelog entry:

* [yoast-components] Adds some changes to `KeywordInput` and `MultiSelect` components, and to CSS, needed for the SEMrush integration feature.

## Relevant technical choices:

* This should be merged together with Yoast/wordpress-seo#16019

## Test instructions

This PR can be tested by following these steps:

* Checkout `feature/semrush` branch in `wordpress-seo`, build against this branch and check that everything's ok.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  * `KeywordInput` component
  * The `MultiSelect` component has been generalised to a `YoastReactSelect` component to add a `SingleSelect` component based on the same code

## UI changes

## Quality assurance


Fixes [P3-161]
